### DISCUSSION TITLE
睿頻

### DISCUSSION THREAD

**Discussion by Ben6uo** [1101-GS4719-C/Python-Community-GS4719-C#391] [General]
應該有不少人在買電腦看到CPU時都看到頻率上寫基頻2.7GHZ  睿頻5GHZ
那當中的睿頻其實是指當程序對CPU資源利用增加時，睿頻加速技術自動開啟，提高CPU頻率，以滿足算力需求
簡單來說就是你現在給的力不足以推動石頭，所以你用更大的力來推動它。

**Comment by pekoranoinu** [comment]
可以幫我解釋睿頻加速技術的原理嗎?

**Comment by boson13579** [comment]
所以睿頻就是超頻ㄇ?
之前在youtube上面看過一堆在用液態氮在玩超頻，可是他們都玩到6G,7G?
是說我電腦沒用神奇的散熱就可以到5.1G,看來intel i7 還是蠻猛的

**Reply by Ben6uo** [reply]
一般認知超頻是進BIOS調
睿頻是自動超但有鎖上限
